src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -72,7 +72,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' != '' and '$(TargetFrameworkVersion)' != ''">
-    <TargetFrameworkMoniker Condition="'$(TargetFrameworkMoniker)' == '' and '$(TargetFrameworkProfile)' != ''">$(TargetFrameworkIdentifier),Version=$(TargetFrameworkVersion),Profile=$(TargetFrameworkProfile)</TargetFrameworkMoniker>
+    <TargetFrameworkMoniker Condition="'$(TargetFrameworkMoniker)' == '' and '$([System.String]::IsNullOrWhitespace($(TargetFrameworkProfile)))' != 'true'">$(TargetFrameworkIdentifier),Version=$(TargetFrameworkVersion),Profile=$(TargetFrameworkProfile)</TargetFrameworkMoniker>
```

Comment:
Should we just add a new `Empty(string)` condition function, alongside `Exists(string)` ?

Reply:
Would replace all the `... '$(foo)' == '' ...` conditions with `... IsEmpty($(foo)) ...`

Reply:
Or maybe even `Defined($(foo))` which would look in the property table of the evaluator to see if it is defined

Reply:
Or better yet, `IsPropertyDefined` and `IsItemDefined`

Reply:
`IsPropertyDefined` wouldn't help for this case: it is defined, but to a bogus value.

I don't think it's worth creating a condition function for this at this point since we have access to the string method, but it's something to consider for the future.

Reply:
Fair enough.